lib/modules/manager/conan/extract.ts

```diff
@@ -64,6 +66,13 @@ export function extractPackageFile(content: string): PackageFile | null {
               replaceString,
               depType,
             };
+            if (matches.groups.revision) {
+              dep.currentDigest = matches.groups.revision;
+              dep.autoReplaceStringTemplate = `{{depName}}/{{newValue}}${userAndChannel}{{#if newDigest}}#{{newDigest}}{{/if}}`;
+              dep.replaceString = `${replaceString}#${dep.currentDigest}`;
+              dep.packageName = `${packageName}#${dep.currentDigest}`;
```

Comment:
```suggestion
```

wrong

Reply:
I did this because I did not find a way to have the `currentDigest` inside `getReleases`. In case we really do not want to have the `currentDigest` I would have to fetch the revision every time `getReleases` is called, which is something we might not want to do.

Reply:
remove, see my `getDigest` suggestion.